src/gui/mzroll/eicwidget.cpp

```diff
@@ -1818,7 +1818,9 @@ QString EicWidget::eicToTextBuffer() {
 void EicWidget::eicToClipboard()
 {
 	if (eicParameters->eics.size() == 0 ) return;
-    getMainWindow()->getAnalytics()->hitEvent("Exports", "Clipboard", 4);
+    getMainWindow()->getAnalytics()->hitEvent("Exports",
+                                              "Clipboard",
+                                              "From Menu");
```

Comment:
@saifulbkhan Would suggest changing the label to "From EIC Menu" to avoid confusion when analysing GA data.